test/relations.test.js

```diff
@@ -568,20 +594,21 @@ describe('relations', function() {
           return new Date();
         }}});
       Address = db.define('Address', {name: String});
-
       Physician.hasMany(Patient, {through: Appointment});
       Patient.hasMany(Physician, {through: Appointment});
       Patient.belongsTo(Address);
       Appointment.belongsTo(Patient);
       Appointment.belongsTo(Physician);
-
       db.automigrate(['Physician', 'Patient', 'Appointment', 'Address'], done);
     });
 
     it('should build record on scope', function(done) {
       Physician.create(function(err, physician) {
+        console.log('============== R-1 :', err, physician);
```

Comment:
Forgotten `console.log`, please remove.

Consider adding `if (err) return done(err)` instead.

If you find the information about `physician` useful, then use `debug` instead.

```js
// at the top (assuming debug is not defined yet)
const debug =require('debug')('test');

// here:
debug('============== R-1 :', err, physician);
```